crimsobot/utils/image.py

```diff
@@ -561,7 +561,9 @@ async def process_image(ctx: Context, image: Optional[str], effect: str, arg: Op
     # grab user image and covert to RGBA
     img = await fetch_image(ctx, image)
 
-    if getattr(img, 'is_animated', False):
```

Comment:
And I already had it sitting there! I am ashamed of myself for missing it before

Reply:
i had a feeling you just missed it :D